x/lazyledgerapp/client/builder.go

```diff
@@ -0,0 +1,172 @@
+package client
+
+import (
+	"context"
+
+	sdkclient "github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/crypto/keyring"
+	sdktypes "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/types/tx"
+	"github.com/cosmos/cosmos-sdk/types/tx/signing"
+	authsigning "github.com/cosmos/cosmos-sdk/x/auth/signing"
+	authtypes "github.com/cosmos/cosmos-sdk/x/auth/types"
+	"github.com/lazyledger/lazyledger-app/app/params"
+	"google.golang.org/grpc"
+)
+
+// Builder wraps the standard sdkclient.TxBuilder add a few quality of life methods along
+// with keeping track of the account number to produce transactions
+type Builder struct {
+	sdkclient.TxBuilder
+	encCfg               params.EncodingConfig
+	address              sdktypes.Address
+	chainID              string
+	accountNum, sequence uint64
+}
```

Comment:
I didn't know what to call this. It's basically just a wrapper of the default transaction builder that focuses on a single account. I don't really like `Builder`, but didn't want to use the same name as `TxBuilder`. I would appreciate any suggestions to make this clearer.

Reply:
I'm not sure yet: in general I think Builder is just fine. Unless this is only for PayForMessage, then I think PayForMessageBuilder could be clearer.